subprojects/core/src/integTest/groovy/org/gradle/api/tasks/NestedInputIntegrationTest.groovy

```diff
@@ -1268,9 +1262,11 @@ class NestedInputIntegrationTest extends AbstractIntegrationSpec implements Dire
         """
 
         expect:
-        // TODO: Remove `requireIsolatedDaemons` once the issue is fixed
-        executer.requireIsolatedDaemons()
         succeeds "crashTask"
+        succeeds "crashTask"
+
+        and:
```

Comment:
💭 I think now the `expect/and` pattern does not fit the test anymore. It could be better to change it to two `when/then` pairs.

Also, `succeeds` does not check whether the task was skipped or actually executed. Given that we add a `skipped` assertion here, it may be good to add a symmetrical on for `executed`.

```
        when:
        succeeds "crashTask"
        then:
        executedAndNotSkipped(":crashTask")

        when:
        succeeds "crashTask"
        then:
        skipped(":crashTask")
```

Reply:
Makes sense, thanks, @alllex.